web/pdf_find_controller.js

```diff
@@ -61,31 +59,30 @@ var CHARACTERS_TO_NORMALIZE = {
 var PDFFindController = (function PDFFindControllerClosure() {
   function PDFFindController(options) {
     this.pdfViewer = options.pdfViewer || null;
-    this.integratedFind = options.integratedFind || false;
-    this.findBar = options.findBar || null;
+
+    this.onUpdateResultsCount = null;
+    this.onUpdateState = null;
 
     this.reset();
 
     // Compile the regular expression for text normalization once.
     var replace = Object.keys(CHARACTERS_TO_NORMALIZE).join('');
     this.normalizationRegex = new RegExp('[' + replace + ']', 'g');
-
-    var events = [
-      'find',
-      'findagain',
-      'findhighlightallchange',
-      'findcasesensitivitychange'
-    ];
-    this.handleEvent = this.handleEvent.bind(this);
-
-    for (var i = 0, len = events.length; i < len; i++) {
-      window.addEventListener(events[i], this.handleEvent);
-    }
   }
 
   PDFFindController.prototype = {
-    setFindBar: function PDFFindController_setFindBar(findBar) {
-      this.findBar = findBar;
+    listenWindowEvents: function PDFFindController_listenWindowEvents() {
+      var events = [
+        'find',
+        'findagain',
+        'findhighlightallchange',
+        'findcasesensitivitychange'
+      ];
+      this.handleEvent = this.handleEvent.bind(this);
```

Comment:
I'm wondering, do we actually need the `this.handleEvent` method now, since it's basically just a wrapper around `this.executeCommand`?
Couldn't we simply remove the method, and use a local function here instead, e.g. something like this:
```javascript
var handleEvent = function (e) {
  this.executeCommane(e.type, e.detail);
}.bind(this);
```